### DISCUSSION TITLE
Links do not always take us to the right place

### DISCUSSION THREAD

**Discussion by madhulikamitra** [abhivaikar/howtheytest#139] [Q&A]
Hi. As i was trying to access a few links, I realised some of them do not take us to the specific blog or info but rather redirect to a parent page - e.g. under stripe - `[Game Day Exercises at Stripe - Learning from kill -9](https://stripe.com/blog/game-day-exercises-at-stripe)` is redirecting me to Blogs in stripe.  Do the links need modification or can i different way?

**Comment by Mihail1Ynakiev** [comment]
./gradlew wrapper --gradle-version <new-version>

**Comment by abhivaikar** [comment]
Hi @madhulikamitra , appreciate your query and apologies for the late response.

There are chances of old articles published by companies becoming unavailable either because the companies decided to unpublish them or moving them to a different website or domain.

I will take a look at what is going wrong with Stripe resources.

Additionally, I am planning to write a scheduled script that will go through each and every resource in this repository and check if any of them are invalid and generate a report of the invalid links so that someone can fix the URLs (or remove them altogether).

Feel free to contribute this script from your side too :)
